CHANGELOG.md

```diff
@@ -17,13 +18,20 @@ Also possible, but for this project less relevant, is `Deprecated` for soon-to-b
 
 ## Unreleased
 
+## SMASH-hadron-sampler-3.1.1
+Date: 2025-02-12
+
+* :sos: Fix that introduces the command line options `./sampler --version` and `./sampler -v` to get the version of the sampler executable
```

Comment:
I would say that this is a ➕ instead of an 🆘. I see your point as we bumped the third digit, but see my comment above.